dashboard/app/models/sections/section.rb

```diff
@@ -30,6 +30,18 @@
 require 'cdo/safe_names'
 
 class Section < ActiveRecord::Base
```

Comment:
It looks like it's still valid to either use `Section.create` and pass in the `login_type` like a normal parameter, or to use `Picture.create` in which I suppose it's inferred.  Are both valid? Do we want to stick to one or the other as a team?  If I `Picture.create!(login_type: 'word')` will I end up with a `Word` without complaint?